spec/requests/menu_items_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'rails_helper'
+
+RSpec.describe 'MenuItems API' do
```

Comment:
Oh wait I noticed you didn't test DELETE

Reply:
Ah good catch

Reply:
Something like:

`# Test suite for DELETE /sellers/:id
  describe 'DELETE /sellers/:id' do
    before { delete "/sellers/#{seller_id}/menu_items/#{id}" }

    it 'deletes the seller' do
       ...
    end 

    it 'returns status code 204' do
      expect(response).to have_http_status(204)
    end
  end`